src/pages/patientView/mutation/MutationInformationContainer.tsx

```diff
@@ -203,7 +203,7 @@ export default class MutationInformationContainer extends React.Component<IMutat
         };
 
         // some columns are dependent on number of samples
-        if (this.props.sampleManager.samples.length > 1) {
```

Comment:
we have to make sure this condition is respected below.  @onursumer recommened that instead of conditionally including the column, we instead use excluded property.  Is this any different with the new table implementation?